implementation/src/main/java/io/smallrye/jwt/build/Jwt.java

```diff
@@ -0,0 +1,37 @@
+package io.smallrye.jwt.build;
```

Comment:
Do new classes to SmallRye projects require a copyright header or do these rely on a root LICENCE file now?

Reply:
@darranl on the root one AFAIK

Reply:
Yes, root LICENCE file